### PR TITLE
feat: LSDV-4941: Support Ranker tag as a hybrid output tag

### PR DESCRIPTION
**Pull request by bmartel**

### CODE REVIEW

File: label_studio_tools/core/label_config.py [hunk 1]
```diff
@@ -63,7 +68,8 @@ def parse_config(config_string):
             conditionals = {}
             if tag.attrib.get('perRegion') == 'true':
                 if tag.attrib.get('whenTagName'):
-                    conditionals = {'type': 'tag', 'name': tag.attrib['whenTagName']}
+                    conditionals = {'type': 'tag',
```
Comment (makseq) [inline label_studio_tools/core/label_config.py:71]: if you want to reduce the length of line, better to do this :
```
conditionals = {
    'type': 'tag', 
    'name': tag.attrib['whenTagName']
}
```

File: label_studio_tools/core/label_config.py [hunk 2]
```diff
@@ -86,13 +106,15 @@ def parse_config(config_string):
...
             continue
+
         parent_name = _get_parent_output_tag_name(tag, outputs)
         if parent_name is not None:
             actual_value = tag.attrib.get('alias') or tag.attrib.get('value')
             if not actual_value:
                 logger.debug(
                     'Inspecting tag {tag_name}... found no "value" or "alias" attributes.'.format(
-                        tag_name=etree.tostring(tag, encoding='unicode').strip()[:50]
+                        tag_name=etree.tostring(
```
Comment (makseq) [inline label_studio_tools/core/label_config.py:116]: let's keep this line untouched?

**Review by makseq — APPROVED** [review]

**Review by makseq — APPROVED** [review]

File: label_studio_tools/core/label_config.py (hunk 2 above)
Comment (bmartel) [reply label_studio_tools/core/label_config.py:116]: Ahhh, the auto linter struck again! Will put back

**Comment by bmartel** [comment]
Closing as the architecture for Ranker has changed, and is now a standard Control tag. So no further requirements to parsing changes.

**Pull request closed without merging**